scanner/image/db/konflux.Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN dnf upgrade -y --nobest && \
     rpm --verbose -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
     rm -rf /var/cache/dnf /var/cache/yum
 
+COPY LICENSE /licenses/LICENSE
```

Comment:
does this file come from the preflight check?

Reply:
No, it is https://github.com/stackrox/stackrox/blob/master/LICENSE